app/src/main/java/com/example/caregiver/Dashboard.java

```diff
@@ -19,14 +19,12 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_dashboard);
+        bottomNavigationView = findViewById(R.id.bottomNavigation);
```

Comment:
Just move this here so we don't run into the same issue toady where the bottom navigation bar got lost during code merges